CMakeLists.txt

```diff
@@ -13,3 +14,35 @@ set (CMAKE_RUNTIME_OUTPUT_DIRECTORY ..)
 add_executable(TransferBench src/TransferBench.cpp)
 target_include_directories(TransferBench PRIVATE src/include)
 
+find_package(ROCM 0.8 REQUIRED PATHS ${ROCM_PATH})
+include(ROCMInstallTargets)
+include(ROCMCreatePackage)
+set(ROCMCHECKS_WARN_TOOLCHAIN_VAR OFF)
+
+set(PACKAGE_NAME TB)
+set(LIBRARY_NAME TransferBench)
+
+rocm_package_setup_component(${PACKAGE_NAME}
+    LIBRARY_NAME ${LIBRARY_NAME}
+    PACKAGE_NAME ${PACKAGE_NAME}
+)
+
+# rocm_install(DIRECTORY ${PROJECT_BINARY_DIR}/src/include
+# DESTINATION include/${PACKAGE_NAME}/build
+# COMPONENT ${LIBRARY_NAME}-headers
+# FILES_MATCHING
+# PATTERN "*.hpp"
+# )
+
+rocm_install(DIRECTORY src/include
+DESTINATION include/${PACKAGE_NAME}
+COMPONENT ${LIBRARY_NAME}
+FILES_MATCHING
+PATTERN "*.hpp"
+)
```

Comment:
As Transferbench only has the single executable, installing the header files shouldn't be necessary. However, you do need to ensure that the executable itself is installed.